docs/UserGuide.adoc

```diff
@@ -495,7 +495,7 @@ You can easily exit Medeina without any mouse click by entering `exit` or `q` in
 
 {empty} +
 
-== Data saving
+== Data Integrity
```

Comment:
I'm not quite sure about this header. Since you didn't add much, the overall content of this section is still data saving.